pkg/apis/events/v1alpha1/pubsub_source_types.go

```diff
@@ -125,9 +126,13 @@ type PubSubSourceStatus struct {
 	// +optional
 	SinkURI string `json:"sinkUri,omitempty"`
 
-	// TransformerURI is the current active transformer URI that has been configured for the GcpPubSubSource.
+	// TransformerURI is the current active transformer URI that has been configured for the PubSubSource.
 	// +optional
 	TransformerURI string `json:"transformerUri,omitempty"`
+
+	// ProjectID is the current in use project ID that has been configured for the PubSubSource.
```

Comment:
This comment seems little funny. 'current in use project ID'.  Is this what gets resolved from the Spec.Project? Or something else.

Reply:
Thanks, fixed.

Reply:
made an issue to make this documented better.